src/modules/shopgate/helpers/basket.php

```diff
@@ -204,6 +206,20 @@ protected function loadOrderBasketPayment(shopgate_oxbasket &$oxBasket, Shopgate
     }
 
     /**
+     * @param shopgate_oxbasket $oxBasket
+     * @param ShopgateCartBase  $shopgateOrder
+     */
+    protected function loadOrderBasketTotal(shopgate_oxbasket &$oxBasket, ShopgateCartBase $shopgateOrder)
```

Comment:
(optional feedback) I know its copy+pasted but shopgate_oxbasket is already a php object and don't needs the & in front of the name. It will be passed by reference per default. Would also rename $oxBasket in $oxidBasket and $shopgateOrder in $shopgateCart.